lib/KCard/__tests__/KCard.spec.js

```diff
@@ -128,4 +128,76 @@ describe('KCard', () => {
       expect(footerSlot.text()).toBe('footer slot content goes here');
     });
   });
+  describe('it renders slot behaviors', () => {
+    it('preserves space when preserve prop is true', () => {
+      const wrapper = makeWrapper({
+        propsData: {
+          to: { path: '/some-link' },
+          title: 'sample title ',
+          headingLevel: 4,
+          preserveAboveTitle: true,
+          preserveBelowTitle: true,
+          preserveFooter: true,
+        },
+        slots: {
```

Comment:
```suggestion
        slots: {
          aboveTitle: '',
          belowTitle: '',
          footer: '',
        },
```



I think that in this test, the most meaningful would be to _not pass_ any content to slots. Because the logic for preserving spaces is designed particularly for cases when there's no content in those slots, so that's the most important thing to test for (and there's no other test right now covering it). And if you rename it to _"preserves space when preserve prop is true and slot is empty"_, it will go very well with it's counter part test _"removes space when preserve prop is false and slot is empty"_ below.

Side note, it's not that this test is wrong, if you'd like to cover every possible combination, then it'd make sense. In that case we'd have two tests instead of one though:
- _"preserves space when preserve prop is true and slot is empty"_
- _"preserves space when preserve prop is true and slot is not empty"_